src/_guides/language/effective-dart/usage.md

```diff
@@ -371,28 +372,27 @@ class Circle {
   }
 
   void _recalculate() {
-    _area = math.PI * _radius * _radius,
+    _area = math.PI * _radius * _radius;
```

Comment:
Found a first syntax error! Which is pretty good, given the number of excerpts in this site!